src/main/java/com/laba/solvd/jdbc/service/interfaces/ISupplierService.java

```diff
@@ -0,0 +1,10 @@
+package com.laba.solvd.jdbc.service.interfaces;
+
+import com.laba.solvd.jdbc.model.Supplier;
+
+import java.util.List;
+
+public interface ISupplierService {
+    Supplier create(Supplier supplier);
+    List<Supplier> getAll();
+}
```

Comment:
please follow same pattern in all interfaces
```suggestion
public interface ISupplierService {

    Supplier create(Supplier supplier);
    List<Supplier> getAll();
    
}
```